### PR TITLE
Feature/minimal nb image testing

### PR DESCRIPTION
**Pull request by lugi0**

### CODE REVIEW

File: run_robot_test.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #/bin/bash
 
-TEST_CASE_FILE=tests/Tests/test.robot
+TEST_CASE_FILE=tests/Tests/test-minimal-image.robot
```
Comment (lugi0) [inline run_robot_test.sh:3]: Need to change back to test.robot

Comment (lugi0) [reply run_robot_test.sh]: Resolved in 4b1d9f5

**Comment by vasukulkarni** [comment]
Lets test via PR test job as well.

**Comment by lugi0** [comment]
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/10/

Test run fails because of the missing images bug, hence I would consider https://issues.redhat.com/browse/RHODS-565 to be a critical bug blocking all my work related to ods-ci

**Review by lugi0 — COMMENTED** [review]
https://opendatascience-jenkins-csb-rhods.apps.ocp4.prod.psi.redhat.com/job/rhods-ci-pr-test/18/console

As expected Jenkins is running test.robot only (and failing) because that file is not updated for v1.0.10, I don't think this is what we want to test though

**Review by vasukulkarni — APPROVED** [review]

**Comment by vasukulkarni** [comment]
can you sign-off your commits?

**Comment by lugi0** [comment]
> can you sign-off your commits?

I'm trying, but that would require rebasing all commits up to now and for some reason I'm having problems doing that. Check the group chat when you can.

**Pull request merged**